README.md

```diff
@@ -56,6 +56,8 @@ The preloader uses the following environment variables to control its behavior:
 
 - `HEAP_PROFILER_TIMELINE_RUN_GC`: Whether or not running Garbage Collector before and after the allocation timeline, to see only remaining objects (default to false).
 
+- `HEAP_PROFILER_LOGGING_DISABLED`: Whether or not disable logging
```

Comment:
Can you please document the value? users will have to set it to "true".

Also can you fix the typo in `HEAP_PROFILER_PROFILE_DURAITON` (DURAITON > DURATION)?